release/Makefile

```diff
@@ -140,7 +140,7 @@ base.txz:
 	    -L /dev/null -N
 	echo "./var/db/etcupdate type=dir uname=root gname=wheel mode=0755" >> ${.OBJDIR}/${DISTDIR}/base.meta
 	sed -n 's,^\.,./var/db/etcupdate/current,p' ${.OBJDIR}/${DISTDIR}/base/var/db/etcupdate/current/METALOG \
-	    >> ${.OBJDIR}/${DISTDIR}/base.meta
+	    | sort >> ${.OBJDIR}/${DISTDIR}/base.meta
```

Comment:
The pipe should be on the previous line. I'd probably rewrap it with a break after the substitution pattern.